.golangci.yml

```diff
@@ -1,110 +1,105 @@
+version: "2"
 run:
   go: "1.23"
-  timeout: 1m
   tests: true
-
-issues:
-  include:
-    - "EXC0014" # Enable revive export rule with "should start with"
-  exclude-rules:
-    # Exclude other linters by default on tests files
-    # we only run testifylint https://github.com/Antonboom/testifylint on test files
-    - path: _test\.go$
-      linters:
-        - errcheck
-        - gci
-        - gofumpt
-        - gosec
-        - gosimple
-        - govet
-        - ineffassign
-        - misspell
-        - staticcheck
-        - unused
-        - usestdlibvars
-        - whitespace
-        - revive
-        - gocritic
-
 linters:
-  disable-all: false
   enable:
-    - errcheck
-    - gci
     - gocritic
-    - gofumpt
     - gosec
-    - gosimple
-    - govet
-    - ineffassign
     - misspell
     - revive
-    - staticcheck
     - testifylint
     - unconvert
-    - unused
     - usestdlibvars
     - whitespace
-
-linters-settings:
-  revive:
+  settings:
+    gocritic:
+      enable-all: true
+      disabled-checks:
+        - appendAssign
+        - hugeParam
+        - rangeValCopy
+        - regexpSimplify
+    govet:
+      disable:
+        - shadow
+        - fieldalignment
+      enable-all: true
+    misspell:
+      locale: US
+    revive:
+      rules:
+        - name: blank-imports
+        - name: context-as-argument
+          arguments:
+            - allowTypesBefore: '*testing.T'
+        - name: context-keys-type
+        - name: dot-imports
+        - name: empty-block
+        - name: error-naming
+        - name: error-return
+        # - name: error-strings
+        - name: errorf
+        # - name: exported
+        #   arguments:
+        #     - checkPrivateReceivers
+        #     - sayRepetitiveInsteadOfStutters
+        - name: increment-decrement
+        - name: indent-error-flow
+        - name: range
+        - name: receiver-naming
+        - name: redefines-builtin-id
+        - name: superfluous-else
+        - name: time-naming
+        - name: unreachable-code
+        - name: var-declaration
+    staticcheck:
+      checks:
+        - all
+        - "-ST1003"
```

Comment:
Add an explanation here, people shouldn't have to do a Google search

I assume this one was in the `comments` presets

Is there so much of it? Can't you add a //nolint in the package that break this rule?

Reply:
Same issue as above, I'll fix them in a future PR